galaxykit/collections.py

```diff
@@ -13,6 +13,12 @@
 from .client import GalaxyClientError
 
 
+def collection_info(client, repository, namespace, collection_name, version):
+    url = f"{client.galaxy_root}content/{repository}/v3/collections/{namespace}/{collection_name}/versions/{version}/"
```

Comment:
not sure that your url needs the leading `{client.galaxy_root}` - the `client.get()` function prepends the galaxy_root to the passed url (see here: https://github.com/ansible/galaxykit/blob/main/galaxykit/client.py#L91)